lib/common/formats.js

```diff
@@ -907,10 +907,11 @@ declare const ${moduleName}: ${JSON.stringify(treeWalker(dictionary.tokens), nul
    */
   'ios-swift/class.swift': function({dictionary, options, file}) {
     const template = _template(
-      fs.readFileSync(__dirname + '/templates/ios-swift/class.swift.template')
+      fs.readFileSync(__dirname + '/templates/ios-swift/swift.template')
     );
     let allTokens;
     const { outputReferences } = options;
+    file['type'] = 'extension';
```

Comment:
I think this needs to be `file['type'] = 'class'`?

Reply:
You are totally right, thanks!